target/linux/ramips/image/mt7621.mk

```diff
@@ -1524,6 +1524,16 @@ define Device/jcg_y2
 endef
 TARGET_DEVICES += jcg_y2
 
+define Device/jdcloud_re-cp-02
+  $(Device/dsa-migration)
+  $(Device/uimage-lzma-loader)
```

Comment:
By the way. lzma-loader may not be necessary for the WiFi6 device with the new bootloader. If not sure, don't touch it.

Reply:
Yes, I'm not familiar with `uimage-lzma-loader`.